scripts/utils.js

```diff
@@ -294,25 +294,26 @@ function attachTooltip (anchor, tooltip, pos = 'right', time = 200) {
     })
 }
 
-function resetExtensionStorage () {
+function initDefaultOptions () {
   chrome.storage.sync.set({
-    agreement: false,
-    show_context: 'tab',
-    resource: false,
-    auto_update_context: false,
-    wm_count: false,
-    auto_archive: false,
-    email_outlinks: false,
     spn_outlinks: false,
     spn_screenshot: false,
```

Comment:
These are 'false' but the 'Email Outlinks' option in 'General' menu is true. But to get an Email, the 'Outlinks' option should be selected, right?

Reply:
Outlinks (if checked in Settings) will only be emailed if the Outlinks checkbox under the SPN button is also checked. We don't want those checked by default because Outlinks takes longer for the wayback machine to save, so want them only if users select them.

Reply:
Thanks for explaining.